third_party/boringssl/BUILD.gn

```diff
@@ -321,6 +321,17 @@ if (!use_cobalt_customizations) {
   }
 }
 
+if (is_starboard) {
```

Comment:
i'd prefer this live in a directory we own so we don't edit upstream's gn file

Reply:
I had a look, I can't come up with a clean and logical place to put it. Unless you want the GN code to live separately from where the tool itself is.

E.g. i'd have to add a deps line into `cobalt/BUILD.gn` 
```
deps = [
   // a bunch of existing .. sandboxes
   "//third_party/boringssl/our_gn_code:boringssl_tool
 ]
 ```
 
And then in that directory, say `third_party/boringssl/our_gn_code/BUILD.gn` , i'd have to do

```
deps = [
  "//third_party/boringssl:boringssl"
]
```

E.g. sort of reverse dependency to parent directory. This becomes more awkward to follow as to what is doing what.

There's also already 22 other mentions of `if (cobalt_customizations)` or `starboard` in this file, i'm not sure i'm making it much worse

Is it worth it ?

Reply:
No, leave as is it's fine